solutions/bronze/usaco-894.mdx

```diff
@@ -5,10 +5,11 @@ title: Grass Planting
 author: Neo Wang, Aadit Ambadkar, Sandeep Wadhwa, Kevin Sheng, Nikhil Chatterjee
 ---
 
-[Official Analysis (C++)](http://www.usaco.org/current/data/sol_planting_silver_jan19.html)
-
-## Video Solution
+<Spoiler title="Hint 1">
+How many types of grass will a node with $x$ neighbors need?
+</Spoiler>
 
+<Spoiler title="Video Solution">
```

Comment:
wait idt we need spoilers for the video sol since they have to click play anyways

Reply:
i think for some videos the thumbnail is some diagram which might give stuff away? we have to keep the styling consistent right?

Reply:
yeah and in #3430 (the template) uh i didn't put a spoiler for the vid sol

Reply:
ah okayy